azure-pipelines.yml

```diff
@@ -83,6 +85,16 @@ steps:
   - task: NodeTool@0
     inputs:
       versionSpec: $(nodeVersion)
+  - task: Bash@3
+    condition: and(eq(variables['Agent.OS'],'Linux'), ge(variables['nodeVersion'], variables['currentNodeVersion']))
+    inputs:
+      targetType: inline
+      script: sudo apt-get install -y libkrb5-dev
+      # nodegit for node>=13.x doesn't come with pre-built binaries, need krb5 to build it during CI
```

Comment:
nit: we should probably move comments above the line of code they refer to for better readability